indexer.go

```diff
@@ -660,8 +660,10 @@ type aliasCommand struct {
 // our response for finding the most recent contact
 type queryResponse struct {
 	Hits struct {
-		Total int `json:"total"`
-		Hits  []struct {
+		Total struct {
+			Value int `json:"value"`
+		} `json:"total"`
```

Comment:
Now an object, this will have repercussions on how we query on the mailroom side.